doc/content/gateways/cotx/COTX-X1_Lite_gateway

```diff
@@ -0,0 +1,51 @@
+---
+title: "COTX X1 Lite"
+description: ""
+---
+
+COTX X1 Lite is a standard LoRaWAN indoor gateway, with a compact appearance, easy to install and deploy. It is highly compatible and has a user-friendly interface. It is rich in software tools, powerful back office management and technical support.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
```

Comment:
Please use the `figure` tag to insert images. If you're not sure how, read the documentation contributing guidelines in this repo, or check .md files of other gateways for an example. This applies to all images in this PR.